hangman.js

```diff
@@ -1 +1,78 @@
-console.log("welcome to hangman");
\ No newline at end of file
+
+
+var welcome = alert("Welcome to Hang My Best Friend! This is your host, Master Bipin and you get to hang your best friend in this show. Let's Begin!!");
+
+var name = prompt ("What is your name?");
+
+
+
+var word = prompt("Hi, " + name + "! You're a shitty friend for wanting to hang your best friend but that's NOM Biz. What's your word?");
+
+var hiddenword = word.split("");
+
+var mask  = word.replace(/./g, " _ ");
+
```

Comment:
You mentioned that you were not able to display correctly guessed letters. By that do you mean that you were not able to replace `_` with letters?

There are a number of ways you could do this. One is to create an array of `_` and replace the appropriate index as letters are correctly guessed. You could also use that `.replace` method you already used, but this time replacing particular characters.

Happy to talk about this more during our 1:1.